pkg/reconcilermanager/controllers/reposync_controller.go

```diff
@@ -329,7 +328,7 @@ func (r *RepoSyncReconciler) Reconcile(ctx context.Context, req controllerruntim
 		}
 	}
 
-	result, err := ComputeDeploymentStatus(deployObj)
+	result, err := kstatus.Compute(unstructuredDeployment)
```

Comment:
Is ComputeDeploymentStatus still used, if not here?

Reply:
We are only using it in nomostest git-server after this change

Reply:
Does nomostest git-server need it? or should it just use kstatus.Compute too?

Reply:
kstatus.Compute can be used directly in  git-server, so removed the ComputeDeploymentStatus and related unit test